package-lock.json

```diff
@@ -86,6 +86,7 @@
       "integrity": "sha512-yDBHV9kQNcr2/sUr9jghVyz9C3Y5G2zUM2H2lo+9mKv4sFgbA8s8Z9t8D1jiTkGoO/NoIfKMyKWr4s6CN23ZwQ==",
       "dev": true,
       "license": "MIT",
+      "peer": true,
```

Comment:
changes in package log seem redundant - there were no package.json changes. It looks like package-log was updated by using older (?) node.js version. My suggestion is to revert this file